app/(candidate)/dashboard/issues/components/ViewModeToggle.js

```diff
@@ -0,0 +1,30 @@
+'use client'
+import { ButtonGroup } from '@mui/material'
+import Button from '@shared/buttons/Button'
+import { FaListUl } from 'react-icons/fa'
+import { TbLayoutGrid } from 'react-icons/tb'
+
+export default function ViewModeToggle({ viewMode, onViewModeChange }) {
```

Comment:
```suggestion
export default function ViewModeToggle({ value, onChange }) {
```

For component props consistency w/ `StatusFilter`, which arguably is the more idiomatic props signature for components that are ultimately inputs of some form or fashion.

Reply:
This still needs addressing